pyamg/strength.py

```diff
@@ -596,6 +629,7 @@ def evolution_strength_of_connection(A, B=None, epsilon=4.0, k=2,
     I = sparse.eye(dimen, dimen, format="csr", dtype=A.dtype)
     Atilde = (I - (1.0/rho_DinvA)*Dinv_A)
     Atilde = Atilde.T.tocsr()
```

Comment:
Yes...this was more a note to myself, because I probably wrote this code.  I don't know why it's transposed and I may remove it.